buildSrc/src/main/kotlin/org/gradle/plugins/buildtypes/BuildType.kt

```diff
@@ -0,0 +1,21 @@
+package org.gradle.plugins.buildtypes
+
+class BuildType(val name: String) {
+    var tasks: List<String> = emptyList()
+    var projectProperties: Map<String,Any> = emptyMap()
+    var active = true
+    var propertiesAction = { properties: Map<String,Any> ->  }
```

Comment:
Just from a readers perspective (someone working in Kotlin over a year), this might read better if declared:
```kotlin
var propertiesAction: (properties: Map<String, Any>) -> Unit = { }
```